wordpress-mobile/WordPress-FluxC-Android
# ThemeStore: Activate theme actions

This PR builds off the work in #542, adding theme activation actions. Note that Jetpack sites require the theme to be installed before activating. You'll have to have at least two themes installed to test on a Jetpack site.

The example app is updated with buttons to test activation. I've also included a single release test, more to come in following PR's.

To test:
* Fire up the example app
* Sign into a WP.com account connected to a Jetpack site
* Type the id of a WP.com theme (for example, `twentyseventeen`) and click `Activate theme WP.com`, verify the theme is activated
* Type the id of an inactive theme that's installed on your Jetpack site and click `Activate theme Jetpack` verify the theme is activated

cc @oguzkocer
